extensions/vertx/deployment/src/main/java/io/quarkus/vertx/deployment/EventConsumerBusinessMethodItem.java

```diff
@@ -1,33 +1,52 @@
 package io.quarkus.vertx.deployment;
 
 import org.jboss.jandex.AnnotationInstance;
-import org.jboss.jandex.MethodInfo;
 
 import io.quarkus.arc.processor.BeanInfo;
+import io.quarkus.arc.processor.InvokerInfo;
 import io.quarkus.builder.item.MultiBuildItem;
 
 public final class EventConsumerBusinessMethodItem extends MultiBuildItem {
 
     private final BeanInfo bean;
     private final AnnotationInstance consumeEvent;
-    private final MethodInfo method;
+    private final boolean blockingAnnotation;
+    private final boolean runOnVirtualThreadAnnotation;
+    private final boolean splitHeadersBodyParams;
```

Comment:
Would you care to add some javadoc that would explain what this actually means?

Reply:
Will do.

Reply:
Done.